sql-to-dbsp-compiler/lib/sqllib/src/types.rs

```diff
@@ -0,0 +1,9 @@
+//! Crate which just re-exports all the types that are used to implement
```

Comment:
This is a module, not a crate. But more importantly I'm not sure it's useful. Anything that needs to be exported should be exported from lib.rs using `pub use ...`.

Reply:
What I want is people who use this stuff to only write one `use` line.
If there's a simple way to do this, great.

Reply:
Right. I don't see how this file helps with this.